components/mixins/profile/profile-mobile-plan-details.schema.json

```diff
@@ -112,6 +114,31 @@
                     "title": "Ported Number",
                     "type": "boolean",
                     "description": "Captures if the customer ports their number from another carrier."
+                  },
+                  "xdm:devices/5GAccess": {
```

Comment:
We do not support / in the property names unless its part of the namespace which is not the case here. Please fix all